tasks/agent.py

```diff
@@ -933,6 +935,30 @@ def omnibus_build(
     with timed(quiet=True) as bundle_elapsed:
         bundle_install_omnibus(ctx, gem_path, env)
 
+    omnibus_cache_dir = os.environ.get('OMNIBUS_GIT_CACHE_DIR')
+    use_omnibus_git_cache = omnibus_cache_dir is not None
```

Comment:
IIUC, `OMNIBUS_GIT_CACHE_DIR` is not set on the Windows Agent builds, meaning that the build cache is not enabled for Windows.

The PR description doesn't mention that this change is Linux-only, so could you confirm that this is expected? If yes, can this be added somewhere in code and in the PR description?